Resources/Prototypes/_LostParadise/Datasets/dreamingverbs.yml

```
@@ -0,0 +1,933 @@
+  - волнующий
+  - обёртывание
+  - разрушение
+  - борьба
+  - извивка
+  - отжим
+  - сочинение
+  - рентгеновский
+  - зевота
+  - крик
+  - застёжка-молния
+  - масштабирование
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)